testsuite/CUDA/cooperative_groups/fail/divergence_thread_block/kernel.cu

```diff
@@ -0,0 +1,22 @@
+//xfail:NOT_ALL_VERIFIED
```

Comment:
How does this test fail? With a data race or barrier divergence? I'm asking this one just for confirmation, so I understand myself what it going on.